src/attackHandler.ts

```diff
@@ -0,0 +1,48 @@
+import { Character } from './character';
+import { WeaponCollision as collision } from './weaponCollision';
+
+export class AttackHandler {
+  static handleAttack(attacker: Character) {
+    if (attacker.weapon === undefined) {
+      return;
+    }
+    if (Date.now() - attacker.attackTimer >= attacker.weapon.cooldown) {
+      attacker.isAttacking = true;
+      attacker.attackTimer = Date.now();
+    }
+  }
+
+  static update(attacker: Character, defenders: Array<Character>) {
```

Comment:
I'd call this `handleAttack` or something.

Reply:
done